lib/racecar/cli.rb

```diff
@@ -102,7 +102,12 @@ def build_parser
         opts.on("-r", "--require STRING", "Require a library before starting the consumer") do |lib|
           $LOAD_PATH.unshift(Dir.pwd) unless load_path_modified
           load_path_modified = true
-          require lib
+          begin
+            require lib
+          rescue => e
+            $stderr.puts "=> Required library failed to load: #{e.message}"
```

Comment:
Can you include the name of the library in the message?

Reply:
Updated this to reference the library name instead of saying "Required library."